src/main/java/com/joyent/manta/monitor/MantaMonitorJerseyServerException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018, Joyent, Inc. All rights reserved.
```

Comment:
Copyright for new files should be 2019. You may want to update your IDE settings.